mars/dataframe/tests/test_core.py

```diff
@@ -320,3 +320,90 @@ def test_key_value(setup):
 
     result = idx.values.execute().fetch()
     np.testing.assert_array_equal(result, raw.values)
+
+
+def test_between(setup):
+    series = Series(pd.date_range("1/1/2000", periods=10))
+    left, right = series[2], series[7]
+
+    result = series.between(left, right).execute().fetch()
+    expected = (series >= left) & (series <= right)
+    expected = expected.execute().fetch()
+    pd.testing.assert_series_equal(result, expected)
+
+
+def test_between_datetime_values(setup):
+    ser = Series(pd.bdate_range("1/1/2000", periods=20).astype(object))
+    ser = ser.execute().fetch()
```

Comment:
Do not execute here, instead, call `.execute().fetch()` on the `result` variable.

Reply:
If I remove line 337, this error appears.
>       ser[::2] = np.nan
E       TypeError: 'Series' object does not support item assignment
Therefore, I have kept line 337.

Reply:
You can create a pandas array, after item assignment, create Mars series.

```
raw = pd.bdate_range(xxx)
raw[::2] = np.nan
ser = Series(raw)
...
```